uchemukolo/EventManager
# Ch 153073280 setup linting

#### What does this PR do?
ESlint is used to find problematic patterns or code that doesn't adhere to the Airbnb style guidelines.
#### Description of Task to be completed?
-npm install eslint --save-dev
-./node_modules/.bin/eslint --init 
-configure Airbnb for style guide
-and test
#### How should this be manually tested?
-create a test.js file and add this line of code var a = 4 
-Then go to the command line and run the command ./node_modules/.bin/eslint test.js you should see the output pointing to the fact that the var keyword is not supposed to be used(according to airbnb style guide).

#### Any background context you want to provide?
none
#### What are the relevant pivotal tracker stories?(if applicable)
CH-#153073280-Setup Linting
